### ISSUE TITLE
Watch on several columns when polling in incrementing mode 

### ISSUE THREAD

**Issue by Thomas-Dangleterre** [SAP/kafka-connect-sap#102]
Hello,

Is there a way to specify multiple columns or a composite primary key when using incrementing mode for a source connector ?
I wanna be able to poll data if one of them are updated. 

Regards.

**Comment by elakito** [comment]
No. But I suppose we could make the incrementing column property to take multiple names and build the where-clause as` col1 >= ... and col2 >= ...` instead of the current single value where-clause `col > ...`.
Contributions are welcome.
